src/components/MintButton/MintButton.tsx

```diff
@@ -36,11 +43,15 @@ export const MintButton: FC<MintButtonProps> = ({ size, ...mintProps }) => {
   const isMintedOut = mintStatus === MintStatus.MintedOut
 
   if ((mintProps.endDate && now >= mintProps.endDate) || isMintedOut) {
+    if (mintProps.address === '0x') {
+      return <ExternalLinkButton partner={mintProps.partnerName} />
```

Comment:
This line will never execute.

The `MintButton` is only used in ternaries with ExternalDrop - or on the trending page with mint.fun drops.

In either case, the drop will never be an external drop at this line

Reply:
@lauchness what should the fallback be here, instead? Just "Mint ended/minted out" ? we can use the message from validate? cc @pedromcunha

Reply:
Just to be clear, are you saying that the address will never be `0x` ? If that's the case then we can simply remove the check and ExternalLinkButton. The CollectButton requires a valid address, otherwise the modal will load infinitely.